Library/Homebrew/cask/audit.rb

```diff
@@ -582,8 +582,9 @@ def check_livecheck_min_os
 
       return if cask_min_os == min_os_string
 
-      min_os_symbol = cask_min_os&.to_sym.inspect || "no minimal OS version"
-      add_error "Upstream defined #{min_os_string.to_sym.inspect} as minimal OS version " \
+      min_os_symbol = cask_min_os&.to_sym.inspect
+      min_os_symbol = "no minimum OS version" if min_os_symbol == "nil"
```

Comment:
How about this? This syntax is more in line with what we do elsewhere

```suggestion
      min_os_symbol = if cask_min_os.present?
        cask_min_os.to_sym.inspect
      else
        "no minimum OS version"
      end
```

Reply:
Much better. Thanks!